uaclient/tests/test_cli_collect_logs.py

```diff
@@ -51,6 +51,7 @@ def test_collect_logs_help(self, _getuid, capsys):
 
     @mock.patch(M_PATH + "tarfile.open")
     # let's pretend all files exist
+    @mock.patch("builtins.open", create=True)
     @mock.patch(M_PATH + "os.path.isfile", return_value=True)
```

Comment:
let's make sure the comment stays in the right place
```suggestion
    @mock.patch("builtins.open", create=True)
    # let's pretend all files exist
    @mock.patch(M_PATH + "os.path.isfile", return_value=True)
```